openedx/features/enterprise_support/api.py

```diff
@@ -586,10 +586,15 @@ def consent_needed_for_course(request, user, course_id, enrollment_exists=False)
         )
 
         if not consent_needed:
+            # TODO: this whole code branch seems to do nothing other than log some misleading info:
```

Comment:
Love to see it. Is it worth having a ticket for investigating/cleaning this up, or do you think it's not that bad?